packages/util-lang/lib/index.mjs

```diff
@@ -0,0 +1 @@
+export * from './isTypeInProtoChain.mjs'
```

Comment:
will the structure be changed in the future (in the case when utils number will be big enough)? or this is acceptale. also the tests should be in the same level as files with the functionality or we can place them in tests folder?

Reply:
> will the structure be changed in the future (in the case when utils number will be big enough)?
I don't suppose it to be too big, so no issues with naming
> also the tests should be at the same level as files with the functionality or we can place them in the tests folder?
good question, adding notes  for #27